packages/services/src/services/VideoPlayer/__tests__/VideoPlayer.test.js

```diff
@@ -0,0 +1,57 @@
+import apiDataResponse from './data/response.json';
+import root from 'window-or-global';
+import VideoPlayerAPI from '../VideoPlayer';
+
+let _scriptLoaded = false;
+
+//jest.mock('../../VideoPlayer');
+
+describe('VideoPlayer', () => {
+  /**
+   *
+   * Mock function for kWidget api and embed objects
+   */
+  const mockFn = () => {
+    root.kWidget = {
+      api: {
+        doRequest: jest.fn(() => Promise.resolve(apiDataResponse)),
+      },
+      embed: jest.fn(() =>
+        Promise.resolve({
+          targetId: 'kaltura_player',
+          wid: '_' + 123456,
+          uiconf_id: 12345678,
+          entry_id: '0_uka1msg4',
+          flashvars: {
+            autoPlay: false,
+          },
+        })
+      ),
+    };
+  };
+  it('should set a loop to check script state is the loaded state or loading state', () => {
+    jest.useFakeTimers();
+    setTimeout(() => {
+      _scriptLoaded = true;
+    }, 500);
+    jest.runAllTimers();
+    expect(_scriptLoaded).toEqual(true);
+  });
```

Comment:
`_scriptLoaded` is a private variable in the service, this test is probably not doing anything.

Reply:
Okay.